contrib/vault/vault-keyring.py

```diff
@@ -65,20 +65,20 @@
 import getpass
 import keyring
 
-import ansible.constants as C
+from ansible.config.manager import ConfigManager, get_ini_config_value
 
 
 def main():
-    (parser, config_path) = C.load_config_file()
-    if parser.has_option('vault', 'username'):
-        username = parser.get('vault', 'username')
-    else:
-        username = getpass.getuser()
+    config = ConfigManager()
+    username = get_ini_config_value(
```

Comment:
LGTM.

Question related to this patch (but not blocking this pr)...
I'm not sure what the best use of the ConfigManager api is.
I had to do something similar for https://github.com/ansible/ansible/pull/27669/files#diff-22b5fc39a68cb5e8e5d54d20f08ac98fR107 

The code here does:

``` python
config = ConfigManager()
username = get_ini_config_value(config._parser, dict(section='vault', key='username'))
```

I used something like the following:
``` python

config_manager = ConfigManager()
username = config_manager.data.get_setting('vault.username')
```

I don't really know which is the preferred usage though.

Reply:
you want `config_manager.get_config_value(...)`